src/pybind/mgr/dashboard/frontend/src/app/shared/enum/icons.enum.ts

```diff
@@ -85,8 +85,12 @@ export enum Icons {
   calendar = 'fa fa-calendar',
   externalUrl = 'fa fa-external-link', // links to external page
   nfsExport = 'fa fa-server', // NFS export
-
+  launch = 'launch',
   /* Icons for special effect */
+  sz16 = '16',
+  sz20 = '20',
+  sz32='32',
+  sz24 = '24',
```

Comment:
since the variable name will be smaller even with the whole text, let's use that.
```suggestion
  size16 = '16',
  size20 = '20',
  size32='32',
  size24 = '24',
```

Reply:
hey @nizamial09  I know some of the things I need to improve here, that I have already noted I just wanted a heads up for the approach I am using as in team there was some discussion before like in future we will be moving to stand alone components. Thats why I raised a PR to check whether this approach works or not.

@ivoalmeida Kindly provide your suggestions too